flask-develop.sh

```diff
@@ -4,4 +4,7 @@ export FLASK_ENV=development
 export FLASK_APP=giftless.wsgi_entrypoint
 export GIFTLESS_DEBUG=1
 
+export GIFTLESS_CONFIG_FILE=${GIFTLESS_CONFIG_FILE:-giftless.yaml}
+echo "Using configuration file: $GIFTLESS_CONFIG_FILE"
+
```

Comment:
Not really tested (just wrote this here) but I think it would be nicer to check if the file exists first and act accordingly:

```suggestion
DEFAULT_CONFIG_FILE=giftless.yaml

if [ -z "$GIFTLESS_CONFIG_FILE" ]; then
    if [ -f "$DEFAULT_CONFIG_FILE" ]; then
        export GIFTLESS_CONFIG_FILE="$DEFAULT_CONFIG_FILE"
        echo "GIFTLESS_CONFIG_FILE not set, defaulting to local config file $DEFAULT_CONFIG_FILE" >&2
    else
        echo "GIFTLESS_CONFIG_FILE not set and $DEFAULT_CONFIG_FILE not found, running with default configuration" >&2
    fi
else
    echo "Using configuration file: $GIFTLESS_CONFIG_FILE" >&2
fi
```

Reply:
@shevron Makes sense, and it works as expected. I pushed a new commit